### PR TITLE
Support hiding base editor

### LINKED ISSUE #682: Support merge view with 3 panels (theirs | final | ours) in addition to the current 4 panels views
In `@jupyterlab/git`, the merge addon from CM5 was used directly to display merge views for plain text using a 3 panels view: `theirs | final | ours`. It will be nice to add a setting in nbdime widget to select either the current 4 panels view or that 3 panel view.

### PR DESCRIPTION
**Pull request by fcollonval**
Fixes #682

It adds a new CLI option `--no-base` to switch from a 4-panels to a 3-panels merge view.

---

Identical with #699 but required after branch rename as it was impossible to check it out on windows

### CODE REVIEW

**Comment by fcollonval** [comment]
bot please update playwright snapshots

**Review by krassowski — APPROVED** [review]
That's still a long branch name ;)

I restarted the flaky test - if it passes this looks good to me.

File: nbdime/args.py [hunk 1]
```diff
@@ -465,6 +465,12 @@ def add_merge_args(parser):
         default=True,
         help="disallow deletion of transient data such as outputs and "
              "execution counts in order to resolve conflicts.")
+    parser.add_argument(
+        '--no-base',
+        dest='show_base',
+        action="store_false",
+        default=True,
+        help="Don't display the base version.")
```
Comment (vidartf) [inline nbdime/args.py:473]: Please add this in `nbdime/webapp/nbmergeweb` instead. It makes no sense for this argument to show up in the CLI command `nbmerge`

File: nbdime/args.py [hunk 2]
```diff
@@ -532,6 +538,7 @@ def args_for_server(arguments):
                 workdirectory='cwd',
                 base_url='base_url',
                 hide_unchanged='hide_unchanged',
+                show_base='show_base',
```
Comment (vidartf) [inline nbdime/args.py:541]: This is mainly a convenience for shared args. For a one-off like `show_base`, it is probably better to add it manually in nbdime/webapp/nbmergeweb.

File: nbdime/config.py [hunk 3]
```diff
@@ -256,6 +256,11 @@ class Merge(_Diffing):
              "execution counts in order to resolve conflicts.",
     ).tag(config=True)
 
+    show_base = Bool(
+        True,
+        help="Whether to show the base version (4-panels) or not (3-panels).",
+    ).tag(config=True)
```
Comment (vidartf) [inline nbdime/config.py:262]: Same here, only needed for web merge.

File: packages/webapp/src/app/merge.ts [hunk 4]
```diff
@@ -54,11 +54,18 @@ function showMerge(data: {
     sanitizer: new Sanitizer(),
   });
 
+  const showBase = getConfigOption('showBase', true);
+
+  if (!showBase) {
+    document.querySelector('#nbdime-header-base')!.textContent = 'Merged';
```
Comment (vidartf) [inline packages/webapp/src/app/merge.ts:60]: Just FYI: this header doesn't seem to be included in UI tests, I assume bc of which node is selected for screenshotting?

**Pull request merged**